src/Microsoft.SqlTools.ServiceLayer/Metadata/MetadataService.cs

```diff
@@ -135,45 +136,40 @@ public void InitializeService(ServiceHost serviceHost)
                     {
                         if (MetadataScriptTempFileStream.IsScriptTempFileValid(connectionInfo.ConnectionDetails.ServerName))
```

Comment:
add comment explaining that if the script temp file is valid, then the scripts don't need to be regenerated (at least that's how I was understanding it)